react/orders101/src/features/navbar/Navbar.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext, useState } from "react";
+import BasketOpen from "../BasketOpen";
+import Hamburger from "../Hamburger";
+import { ShoppingCartContext } from "../../App";
+import Cart from "../cart/Cart";
+
+function Navbar() {
+  let [isCartOpen, setIsCartOpen] = useState(false);
+
+  const closeCart = () => {
+    setIsCartOpen(false);
+  };
+  const openCart = () => {
+    setIsCartOpen(true);
+    console.log(isCartOpen);
+  };
+
+  return (
+    <React.Fragment>
+      <div class="nav container">
+        <div class="hamburger__btn-open">
+          <Hamburger />
+        </div>
+        <a class="nav__icons" href="#stores">
```

Comment:
This would be another example of where we could iterate over mock data

Reply:
skipped for now